seedlet/assets/sass/abstracts/_config.scss

```diff
@@ -65,4 +65,7 @@ $typescale-ratio: 1.2; // Run ratio math on 1em == $typescale-base * $typescale-
 	--global--border-radius-lg: #{$typescale-root};
 	--global--border-radius-pill: #{10 * $typescale-root};
 	--global--border-radius-none: 0;
+
+	/* WordPress Variables */
+	--wp--style--color--link: var(--global--color-primary);
```

Comment:
Defining this here seems a little weird to me since it doesn't match our other rule naming, but I wasn't sure of how to handle this otherwise — we need to define this value otherwise we'll end up with the WordPress default. 

I also needed to switch to using the `var(--wp--style--color--link)` color in our `a` rules to get this to work. I guess that's ok, since we'll need to do that in the future anyway.

Reply:
Left a suggestion below that wouldn't impact this file.

But yeah, this gets at a discussion of how we will eventually handle mapping of presets from the theme config to the set of design tokens.

Reply:
Yeah. For the block-based version of this, I bundled those in a totally separate stylesheet like this: 

https://github.com/Automattic/themes/blob/feaf746c0f6d1ef9297cce069ff7a22c4eb79e90/seedlet-blocks/style.css#L33-L49

That was really clean, but probably doesn't make sense here.